README.md

```diff
@@ -44,4 +44,22 @@ You may need to use gcloud to provide GCR
 
 ```
 local-dev/setup_local_env.sh <environment>
+```
+
+## Testing
```

Comment:
hmm, should this go in the local_dev README instead? don't have a strong opinion.

Reply:
Feels the WM way has to many readmes and hard to follow. So I am thinking about to have one readme and will have local server, deploy to gke, how to run test sections. Removed local_dev/ readme and put everythinh here.